app/controllers/casa_cases_controller.rb

```diff
@@ -6,7 +6,14 @@ class CasaCasesController < ApplicationController
   # GET /casa_cases
   # GET /casa_cases.json
   def index
-    @casa_cases = policy_scope(current_organization.casa_cases)
+    org_cases = if current_user.volunteer?
+                  current_organization.casa_cases
+                else
+                  # to avoid N + 1 error raised by bullet
+                  CasaOrg.includes(:casa_cases).find_by(id: current_user.casa_org_id).casa_cases
+                end
```

Comment:
@mecastelom It's not clear to me why this conditional (`if`/`else`) is needed. Does the user being a volunteer affect what queries are run for this controller/view? I looked at the corresponding view, but couldn't see anything offhand.

Reply:
@alindeman There is casa_case_policy 2 that avoid the n+1 issue for volunteers.

Reply:
@mecastelom Gotcha. Do you think the `includes` could be moved into the policy? That might be a more centralized place to fix it.

Reply:
Or alternatively, would it be OK to add the `includes` to both cases here, removing the conditional? I'm mostly concerned that this looks a lot like policy being enforced in the controller, which we generally don't like. It's not _actually_, but it looks a lot like it and it might accidentally evolve into that over time if we're not careful. The branching logic also adds an additional conditional that must be maintained and tested.